paddle/fluid/operators/sum_op.cc

```diff
@@ -141,6 +133,18 @@ class SumOp : public framework::OperatorWithKernel {
       PADDLE_ENFORCE_NE(dtype, -1,
                         "Sum operator should have at least one tensor");
 
+#ifdef PADDLE_WITH_MKLDNN
+      if (library == framework::LibraryType::kPlain &&
+          platform::CanMKLDNNBeUsed(ctx)) {
+        if (static_cast<framework::proto::VarType::Type>(dtype) ==
+            framework::proto::VarType::FP32) {
```

Comment:
This condition should match the one from the original `paddle/fluid/operators/mkldnn/sum_mkldnn_op.cc:62`.

Reply:
Please, join the three `if`s into a single one.

Reply:
> Please, join the three `if`s into a single one.

Thank you very much. I have fixed these two reviews. please take a look.